text/pdf_formatted_text.go

```diff
@@ -50,6 +50,12 @@ func main() {
 		log.Fatalf("Error: %v", err)
 	}
 
+	// Generate styled paragraph text wrapping and overflow subchapter.
+	err = styledParagraphTextOverflow(c, chap, fontRegular, fontBold)
```

Comment:
There seems to be something wrong in this example (by looking at the output file). The content of the last chapter should fit in the available space but instead it is placed on a new one. I'm not sure what causes this (might be a bug regarding the margins of the chapter component). Does the whole thing fit on one page if you control the margins only from the paragraphs below the chapter/subchapter titles?

Reply:
Is that a bug in the example or unipdf?

Reply:
It's a problem with the example, it's bottom margin is set to 20.